pkg/asset/ignition/content/bootkube.go

```diff
@@ -82,7 +82,6 @@ then
 		--manifest-etcd-serving-ca=etcd-client-ca.crt \
 		--asset-input-dir=/assets/tls \
 		--asset-output-dir=/assets/kube-apiserver-bootstrap \
-		--config-override-file=/usr/share/bootkube/manifests/config/config-overrides.yaml \
```

Comment:
@sttts if we don't specify them here they will default I think and this command will fail (haven't tested it yet), but I think we should first remove the defaults in our image repos and then this should be green?